utilities/checkpoint/checkpoint_impl.cc

```diff
@@ -377,12 +377,12 @@ Status CheckpointImpl::ExportColumnFamily(
         live_file_metadata.db_path = export_dir;
         live_file_metadata.smallest_seqno = file_metadata.smallest_seqno;
         live_file_metadata.largest_seqno = file_metadata.largest_seqno;
-        live_file_metadata.smallestkey = std::move(file_metadata.smallestkey);
-        live_file_metadata.largestkey = std::move(file_metadata.largestkey);
```

Comment:
Why do we stop populating smallestkey and largestkey?

Reply:
It's not used in `CreateColumnFamilyWithImport()`. But I decided to add them back for completeness and in case they are used somewhere else by user.